fdbkubernetesmonitor/monitor.go

```diff
@@ -397,6 +406,19 @@ func (monitor *Monitor) WatchConfiguration(watcher *fsnotify.Watcher) {
 	}
 }
 
+// handleFileChange will perform the required action based on the changed/modified file.
+func (monitor *Monitor) handleFileChange(changedFile string) {
+	if changedFile == fdbClusterFilePath {
+		err := monitor.PodClient.updateFdbClusterTimestampAnnotation()
+		if err != nil {
+			monitor.Logger.Error(err, fmt.Sprintf("could not update %s annotation", ClusterFileChangeDetectedAnnotation))
+		}
+		return
```

Comment:
why not go ahead with the `monitor.LoadConfiguration()` even if we miss signalling the change on annotation? wouldn't we miss the signal and require an extra change?

Reply:
discussed offline: I think the problem here is more that there is no retry mechanism in the case of a failure to update annotations (or in general)
I think this is fine as-is but that this error should get retried when that is added (separate PR)